src/main/groovy/com/palantir/gradle/docker/DockerRunPlugin.groovy

```diff
@@ -123,7 +123,10 @@ class DockerRunPlugin implements Plugin<Project> {
                     args.add("${localFile.absolutePath}:${volume.value}")
                 }
                 args.addAll(ext.env.collect{ k, v -> ['-e', "${k}=${v}"] }.flatten())
-                args.addAll(['--name', ext.name, ext.image])
+				for (String argument : ext.arguments) {
+					args.add(argument)
+				}
+				args.addAll(['--name', ext.name, ext.image])
```

Comment:
nit: add this argument before adding the custom arguments

Reply:
Hi @pkoenig10 , I updated this section to specify --name before custom arguments, but ext.image must still come after the custom arguments, as per the docker run manual, the name of the image should come after all of the flags, but before the actual command being executed.